unit/solvers/smt2_incremental/smt_bit_vector_theory.cpp

```diff
@@ -60,6 +60,59 @@ TEST_CASE("SMT bit vector extract", "[core][smt2_incremental]")
   }
 }
 
+TEST_CASE("SMT bit vector bitwise operators", "[core][smt2_incremental]")
+{
+  const smt_bit_vector_constant_termt two{2, 8};
+  const smt_bit_vector_constant_termt three{3, 8};
+  const smt_bit_vector_constant_termt wider{4, 16};
+  const smt_bool_literal_termt true_val{true};
+  SECTION("not")
+  {
+    const auto function_application = smt_bit_vector_theoryt::make_not(two);
+    REQUIRE(
+      function_application.function_identifier() ==
+      smt_identifier_termt("bvnot", smt_bit_vector_sortt{8}));
+    REQUIRE(function_application.get_sort() == smt_bit_vector_sortt{8});
+    REQUIRE(function_application.arguments().size() == 1);
+    REQUIRE(function_application.arguments()[0].get() == two);
+
+    cbmc_invariants_should_throwt invariants_throw;
+    REQUIRE_THROWS(smt_bit_vector_theoryt::negate(true_val));
```

Comment:
Why is it a `negate` here? Should this be an `make_not` instead?

Reply:
Because "copy-pasta". Yes, well spotted. This should now be fixed.